src/components/ItemsList.js

```diff
@@ -15,6 +15,11 @@ const wasItemPurchasedWithinLastOneDay = (lastPurchasedOn) => {
 };
 
 const ItemsList = () => {
+  const [searchTerm, setSearchTerm] = React.useState('');
```

Comment:
Great job team!
Yes I feel the same way about this too, since we are already importing useState like this `import React, { useState } from 'react';` in the other file

Reply:
haha I see, I should have take a closer look at this, of course we are importing it, and not writing it this way ;) Sorry for not taking a closer look myself before PR.